src/Grid/Column/Resolver/Metadata/DocumentResolver.php

```diff
@@ -47,9 +48,13 @@ public function resolve(Column $column, ElementInterface $element): ColumnData
             return $this->getColumnData($column, null);
         }
 
-        $documentIndex = $this->documentService->getDocument(
-            reset($document['document'])
-        );
+        try {
+            $documentIndex = $this->documentService->getDocument(
```

Comment:
should we not call this $relatedDocument just for consistency? :)

Reply:
Yeah sure makes sense :+1: